scraper.py

```diff
@@ -69,6 +70,8 @@
 
 CHAPTERS_URL_TPL = "https://feeds.fireside.fm/{show}/json/episodes/{ep_id}/chapters"
 
+# Regex to strip Episode Numbers and information after the |
```

Comment:
I personally like to link to regex101.com to help people that want to know more about how the regex works.

```suggestion
# Regex to strip Episode Numbers and information after the |
# https://regex101.com/r/gkUzld/
```

If people go to PCRE2 it'll actually show you how it matches with the debugging feature at the bottom left :grin: